activesupport/lib/active_support/cache/mem_cache_store.rb

```diff
@@ -117,15 +122,19 @@ def initialize(*addresses)
         super(options)
 
         unless [String, Dalli::Client, NilClass].include?(addresses.first.class)
-          raise ArgumentError, "First argument must be an empty array, an array of hosts or a Dalli::Client instance."
+          raise ArgumentError, "First argument must be an address or array of addresses."
```

Comment:
```suggestion
          raise ArgumentError, "First argument must be an empty array, address, or array of addresses."
```

Sorry, this was [my misunderstanding](https://github.com/rails/rails/pull/47340#discussion_r1102859342). Empty array is fine.

Reply:
:phew: I had started adding a test to not allow it and saw we can use the default hahaha, thanks for circling back 👍